### PR TITLE
Added loggers to connections controllers

### PR DESCRIPTION
**Pull request by Nabeelcodes110**
addConnections:
-removed check for valid user(user can only send req to valid users)
-added guard clause for response3

getConnections:
-removed the getPrivate Message call for making new API for get messages
-removed sortConnection function

rejectConnection : 
-updated the Query for getPendingRequest as only the reciever can reject the request

withdrawRequest : 
-updated the query = only sender can withdraw the request

Tested on Local server everything working.

### CODE REVIEW

**Comment by Nabeelcodes110** [comment]
addIdea : 
-removed the if block for checking previously created team.

deleteIdea :
removed if block for checking the id of idea. will add the sanitizer for the same later.

getIdea : 
-removed if block to check if the idea exist or not

File: src/controllers/connection.controller/getConnection/index.js [hunk 1]
```diff
@@ -1,38 +1,27 @@
...
-    return res.json({
-      sucess: false,
-      errorCode: 'InternalServerError',
-      errorMessage: JSON.stringify(response2.errors),
-      data: null,
-    });
+  const connections = response1.result.data.connections;
 
-  const connections = sortConnections(response1.result.data.connections, response2.result.data.private_messages, user_id);
+  connections.sort((a, b) => b.id - a.id);
```
Comment (Shivz3232) [inline src/controllers/connection.controller/getConnection/index.js]: Why are you sorting by connection id?

File: src/controllers/connection.controller/rejectConnection/queries/queries.js [hunk 2]
```diff
@@ -6,16 +6,14 @@ const getUserId = `query getUser($cognito_sub: String_comparison_exp) {
...
       user1
       user2
       status
     }
-  }
-  `;
+  }`;
 
 const deleteConnection = `mutation rejectConnection($user1: Int, $user2: Int) {
     delete_connections(where: {_or: [{_and: [{user1: {_eq: $user1}}, {user2: {_eq: $user2}}]}, {_and: [{user1: {_eq: $user2}}, {user2: {_eq: $user1}}]}], status: {_eq: "pending"}}) {
```
Comment (Shivz3232) [inline src/controllers/connection.controller/rejectConnection/queries/queries.js]: Shouldn't we also remove the or condition from the delete connection query for the same reason?

File: src/controllers/idea.controller/deleteIdea/index.js [hunk 3]
```diff
@@ -1,26 +1,19 @@
+const logger = require('../../../config/logger');
 const catchAsync = require('../../../utils/catchAsync');
 const { query: Hasura } = require('../../../utils/hasura');
 const { delete_idea } = require('./queries/queries');
 
 const deleteIdea = catchAsync(async (req, res) => {
-  const id = await req.body.id;
+  const id = req.body.id;
```
Comment (Shivz3232) [inline src/controllers/idea.controller/deleteIdea/index.js:7]: Why did you remove the check? Will you add it back when adding the sanity checks?
